Dockerfile

```diff
@@ -1,13 +1,16 @@
 FROM ubuntu:20.04
 LABEL maintainer="rubenperezm"
 
+ARG BRANCH
+
 WORKDIR /app
 SHELL ["/bin/bash", "-c"]
 
 # Git clone the repository
 RUN apt-get update && apt-get install -y git
 RUN git clone https://github.com/vre-hub/reana-jupyterlab-extension.git
```

Comment:
This solution works very well, indeed. However, it isn't a good practice to make calls to git.
What about doing something like (for example)

COPY . /reana_jupyterlab_extension
WORKDIR /reana_jupyterlab_extension


This would also solve the cases in where we are working from any branch, as the corresponding checkout stage will fetch the correct branch/status

I think that you'd need to slightly adapt the `Dockerfile` too, thanks !

Reply:
Changed